src/pages/Admin/Venue/DetailsValidationSchema.ts

```diff
@@ -46,53 +48,34 @@ const createFileSchema = (name: string, required: boolean) =>
       }
     );
 
-const urlIfNoFileValidation = (fieldName: string) =>
-  Yup.string().when(
-    fieldName,
-    (file: FileList | undefined, schema: Yup.MixedSchema<FileList>) =>
-      file && file.length > 0
-        ? schema.notRequired()
-        : schema.required("Required")
-  );
-
 export const validationSchema = Yup.object()
   .shape<VenueInput>({
     template: Yup.mixed<VenueTemplate>().required(),
-    name: Yup.string()
-      .required("Venue name is required")
-      .min(
-        VENUE_NAME_MIN_CHAR_COUNT,
-        ({ min }) => `Name must be at least ${min} characters`
-      )
-      .max(
-        VENUE_NAME_MAX_CHAR_COUNT,
-        ({ max }) => `Name must be less than ${max} characters`
-      )
-      .when(
```

Comment:
[note] This is exactly the same as the version in `src/pages/Admin/Details/ValidationSchema.ts` that is now imported here.